ext/standard/dl.c

```diff
@@ -85,7 +111,7 @@ PHPAPI int php_load_extension(char *filename, int type, int start_now)
 	zend_module_entry *module_entry;
 	zend_module_entry *(*get_module)(void);
 	int error_type, slash_suffix;
-	char *extension_dir;
+	char *extension_dir, *err1, *err2;
```

Comment:
you can define 'err1' or 'err2', use 'err_one', 'err_two' instead! 
What do you think of it?